processor/src/lib.rs

```diff
@@ -5,8 +5,9 @@
 extern crate alloc;
 
 pub use vm_core::{
-    chiplets::hasher::Digest, crypto::merkle::MerkleError, errors::InputError, Kernel, Operation,
-    Program, ProgramInfo, StackInputs, StackOutputs, Word,
+    chiplets::hasher::Digest, crypto::hash::blake::Blake3_192, crypto::hash::rpo::Rpo256,
+    crypto::merkle::MerkleError, errors::InputError, Kernel, Operation, Program, ProgramInfo,
+    StackInputs, StackOutputs, Word,
```

Comment:
I'd probably put hash function exports under a separate module. Maybe something like `crypto::hashes` - but we can do this later too.

Reply:
I did a structure like that in `vm-core`; lemme know your thoughts